packages/syft/src/syft/service/attestation/attestation_service.py

```diff
@@ -0,0 +1,60 @@
+# stdlib
+from collections.abc import Callable
+
+# third party
+import requests
+
+# relative
+from ...serde.serializable import serializable
+from ...store.document_store import DocumentStore
+from ..context import AuthedServiceContext
+from ..response import SyftError
+from ..response import SyftSuccess
+from ..service import AbstractService
+from ..service import service_method
+from ..user.user_roles import GUEST_ROLE_LEVEL
+from .attestation_constants import ATTESTATION_SERVICE_URL
+from .attestation_constants import ATTEST_CPU_ENDPOINT
+from .attestation_constants import ATTEST_GPU_ENDPOINT
+
+
+@serializable()
+class AttestationService(AbstractService):
+    """This service is responsible for getting all sorts of attestations for any client."""
+
+    def __init__(self, store: DocumentStore) -> None:
+        self.store = store
+
+    def perform_request(
+        self, method: Callable, endpoint: str, raw: bool = False
+    ) -> SyftSuccess | SyftError | str:
+        try:
+            response = method(f"{ATTESTATION_SERVICE_URL}{endpoint}")
+            response.raise_for_status()
+            message = response.json().get("result")
+            raw_token = response.json().get("token")
+            return raw_token if raw else SyftSuccess(message=message)
```

Comment:
Leaving a Note, even during the situations, when the output if False, it is wrapped in SyftSuccess message, in future it would be good to wrap it in Syft Error.

Reply:
Added this change.